code_map.py

```diff
@@ -745,18 +732,18 @@ class show_code_map(sublime_plugin.TextCommand):
     # -----------------
 
     def run(self, edit):
-        global ACTIVE, CURRENT_TEMP_ID, TEMP_VIDS, TEMP_VIEWS
+        global ACTIVE, CURRENT_TEMP_ID, TEMP_VIDS, TEMP_VIEWS, CUSTOM_MAPPERS
 
         w = win()
         Mapper.block_max_pane(True)
         groups = w.num_groups()
         current_view = self.view
         map_view = get_code_map_view()
 
-
         if not map_view:            # opening Code Map
 
             ACTIVE = True
+            CUSTOM_MAPPERS = os.listdir(path.join(sublime.packages_path(), 'User', 'CodeMap', 'custom_mappers'))
 
```

Comment:
Reading the custom mappers here too, so restarting ST isn't needed after adding a new custom mapper.

Reply:
Good. Not sure if removing a custom mapper file will have any runtime effect. Buts since there is no promise in the documentation of the _true dynamic loading_ we are fine.